website/client/src/components/admin-panel/index.vue

```diff
@@ -0,0 +1,754 @@
+<template>
+  <div
+    v-if="user.contributor.admin"
+    class="row standard-page"
+  >
+    <div class="well">
+      <h1>Admin Panel</h1>
+      <div class="row">
+        <div class="form-inline col-12">
+          <div class="form-group">
+            <input
+              v-model="heroID"
+              class="form-control"
+              type="text"
+              :placeholder="'User ID or Username; blank for your account'"
+              :style="{ 'min-width': '45ch' }"
+              @keyup.enter="loadHero(heroID)"
+            >
+          </div>
+          <div class="form-group">
+            <button
+              class="btn btn-secondary"
+              @click="loadHero(heroID)"
+            >
+              Load User (click or hit enter)
+            </button>
+          </div>
+        </div>
+      </div>
+
+      <div
+        v-if="hero && hero.profile"
+        class="row"
+      >
+        <div class="form col-12">
+          <h2>@{{ hero.auth.local.username }} &nbsp; / &nbsp; {{ hero.profile.name }}</h2>
+          <div class="form-group">
+            {{ hero._id }} &nbsp;
+            <router-link :to="{'name': 'userProfile', 'params': {'userId': hero._id}}">
+              profile link
+            </router-link>
+          </div>
+
+          <div class="accordion-group">
+            <h3
+              class="expand-toggle"
+              :class="{'open': expandPriv}"
+              @click="expandPriv = !expandPriv"
+            >
+              Privileges, Gem Balance
+            </h3>
+            <div v-if="expandPriv">
+              <div class="form-group">
```

Comment:
this should probably be a form

Reply:
Done in 26fdcbbee51937d39a149a20bf7ccd1cf674ca18

Reply:
Done.